jwsig/jws.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2024 OpenPubkey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package jwsig
```

Comment:
Thanks for putting this together

Use the `git mv` command to move the files. This will preserve the git history

Reply:
Got it, but what will be the source file name and destination file in the git mv command? Is it just `git mv oidc jwsig`?

Reply:
Yeah, that should work. If it doesn't checkout https://git-scm.com/docs/git-mv

Reply:
@EthanHeilman, I tried following the docs but I am still ending up with 
```
$ git mv oidc jwsig
fatal: bad source, source=oidc, destination=jwsig/oidc
```
Can you help me figure out why this persists?

Reply:
what directory are you?

Reply:
jwsig